Library/Formula/speedread.rb

```diff
@@ -0,0 +1,19 @@
+class Speedread < Formula
+  desc "Simple terminal-based rapid serial visual presentation (RSVP) reader"
+  homepage "https://github.com/pasky/speedread"
+  url "https://github.com/pasky/speedread/archive/6d1c5ebf276af5432ae9639cdd5f452c00c5e888.tar.gz"
```

Comment:
I asked the upstream maintainer to add a version tag. The last commit to the project is 2014 so I don't know whether this is gonna happen.